main.go

```diff
@@ -505,6 +558,30 @@ func getDefaultPort() (port string, err error) {
 		portPath = "/dev/cu.usb*"
 	case "linux":
 		portPath = "/dev/ttyACM*"
+	case "windows":
+		cmd := exec.Command("wmic",
+			"PATH", "Win32_SerialPort", "WHERE", "Caption LIKE 'USB Serial%'", "GET", "DeviceID")
```

Comment:
Ugh that looks a lot like SQL, but whatever...

Reply:
That is correct: https://docs.microsoft.com/en-us/windows/win32/wmisdk/wmi-and-sql